File: hw/nacato/u3/hw26Observer/Observer.py/observer/view/application.py
from model.investor import Investor
from model.ibm import IBM


def main():

    s = Investor("Sorros")
    b = Investor("Berkshire")

    ibm = IBM("IBM", 120.00)

    ibm.add_observer(s)
    ibm.add_observer(b)

    ibm.set_price(120.10)
    ibm.set_price(121.00)
    ibm.set_price(120.50)
    ibm.set_price(120.75)

    ibm.set_symbol("IBMTEST")


if __name__ == "__main__":
    main()

File: hw/chavez/u3/hw27Observer/JavaScript/view/Application.js
import IBM from "../model/IBM.js";
import Investor from "../model/Investor.js";

const sorros = new Investor("Sorros");
const berkshire = new Investor("Berkshire");

const ibm = new IBM("IBM", 120.00);

ibm.addObserver(sorros);
ibm.addObserver(berkshire);

ibm.setPrice(120.10);
ibm.setPrice(121.00);
ibm.setPrice(120.50);
ibm.setPrice(120.75);

ibm.setSymbol("IBMTEST");